src/DipoleAlgorithms.cc

```diff
@@ -137,34 +137,54 @@ void NaiveDipoleCentroid::measure(
 ) const {
     afw::detection::PeakCatalog const& peaks = source.getFootprint()->getPeaks();
 
-    naiveCentroid(source, exposure, peaks[0].getI(), (peaks[0].getPeakValue() >= 0 ?
-                                                       getPositiveKeys() :
-                                                       getNegativeKeys()));
-    if (peaks.size() > 1) {
-        naiveCentroid(source, exposure, peaks[1].getI(), (peaks[1].getPeakValue() >= 0 ?
-                                                           getPositiveKeys() :
-                                                           getNegativeKeys()));
+    /*
```

Comment:
Please don't create a branch for this.  This looks like a classic case for using Trace logging ( @hchiang2 )